op-challenger/config/config_test.go

```diff
@@ -132,120 +169,120 @@ func TestCannonRequiredArgs(t *testing.T) {
 		traceType := traceType
 
 		t.Run(fmt.Sprintf("TestCannonBinRequired-%v", traceType), func(t *testing.T) {
```

Comment:
Can you add some new unit tests for the new errors?  So for `VmBin` and `Server` they would set non-empty values that point to non-existent files.  For each vm type - so 6 new tests in total.  You can nest them inside these `Test<VM>RequiredArgs` tests.

Reply:
Done in dd1d218168b52d1e8fe3c9e2233aa8fd19b8f81f.